### DISCUSSION TITLE
What might be wrong with this address?

### DISCUSSION THREAD

**Discussion by brad-4k** [osm-search/Nominatim#2880] [Geocoding around the world]
I'm curious as to why this address doesn't resolve.  I don't know the format for addresses in France which may be part of the problem:

5000 La Ribeyre,Lavoûte-Chilhac,FR 43380

**Comment by brad-4k** [comment]
Not sure if I should start another thread but here is another address that doesn't resolve correctly:

1800 S Kirkman Rd,Orlando,US 32811

**Comment by mtmail** [comment]
Please do a bit of self research first. Is the street on the OpenStreetMap map, is the house number? Nominatim can only find what's in OpenStreetMap data. We can't add addresses based on user reports. https://www.openstreetmap.org/#map=16/45.1509/3.4045

**Reply by brad-4k** [reply]
Thank you for your reply.  I understand and do try to validate the addresses using both the Openstreetmap map and Google.

I'm trying to validate (in a loose way) Nominatim (and in some respects OpenStreetmap) against a database we have of 100K addresses (with associated lat/lon).  I'm just trying to gauge how much may be user (or my data error) versus issues with OpenStreetmaps (and by extension Nominatim).   I understand there are a LOT of points of error (on my side) but, again, just trying to make sure I can reduce the errors imposed by me providing malformed addresses so I can get an idea on the number of bad data (on my side) versus missing data (on the OpenStreetmap side).

Here is a snapshot of testing 1000 of the over 100K addresses.  Large circles represent the source position, small circles represent the Nominatim answer(s), lines connect them.  The criteria I have for placing a circle is if there was no location found by Nominatim, or if the distance is more than 0.5km away.  Simply an example, not the actual results.

<img width="1131" alt="Screen Shot 2022-11-14 at 11 49 58 AM" src="https://user-images.githubusercontent.com/78495562/201752349-ffe00ef6-558b-42cc-a571-1d5aeaae56c5.png">
